uaclient/cli.py

```diff
@@ -327,6 +327,7 @@ def _perform_enable(
     @return: True on success, False otherwise
     """
     ent_cls = entitlements.ENTITLEMENT_CLASS_BY_NAME[entitlement_name]
+    allow_beta |= cfg.cfg.get("features", {}).get("allow_beta", False)
```

Comment:
Let's be a bit more defensive here and get allow_beta and make sure it is booleanish ("true" | "false" | "True" | False) and raise UserFacinError if not since this might be customer visible (and ripe with potential to misspell or provide invalid values).